### DISCUSSION TITLE
Welcome to Liquid-Pull-To-Refresh Discussions!

### DISCUSSION THREAD

**Discussion by aagarwal1012** [aagarwal1012/Liquid-Pull-To-Refresh#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnshiver** [comment]
Am curious, what are some ideas to inform the user that pull to refresh is available? In my app its not obvious, but very useful
